modules/structured_light/include/opencv2/structured_light/slmono_calibration.hpp

```diff
@@ -0,0 +1,59 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef OPENCV_structured_light_mono_calibration_HPP
+#define OPENCV_structured_light_mono_calibration_HPP
+
+#include <opencv2/core.hpp>
+//#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc.hpp>
+#include "opencv2/core/utility.hpp"
+#include <opencv2/calib3d.hpp>
+
+using namespace std;
```

Comment:
`using namespace` is not allowed in OpenCV public headers (to avoid breaking of users code)